### ISSUE TITLE
Docker create new container even when state=started

### ISSUE THREAD

**Issue by ansibot** [ansible/ansible#30058]
##### ISSUE TYPE

Bug Report

##### COMPONENT NAME

docker module

##### ANSIBLE VERSION

2.1

##### SUMMARY

From @bocharsky-bw on 2015-08-19T13:38:38Z
Try to describe problem with next example.

I create new container manually and name it as `elasticsearch.container`. Then I push some data to elasticsearch index.

When I do `$ docker stop elasticsearch.container` and then `$ docker start elasticsearch.container` - my data in this container restore and I could use it after container starts.

If I do the same things with ansible playbook, ansible recreate new container when it stops. I mean, if I do `$ docker stop elasticsearch.container` and then run playbook with next instructions:

``` yaml
  tasks:
    - name: elasticsearch container
      docker:
        image: elasticsearch:latest
        name: elasticsearch.container
        state: started
        ports:
          - "9200:9200"
          - "9300:9300"
```

I have new container without my previous data. I want to emphasize attention on `state: started`.

Does it normal `docker_module` behavior for this state? And why my data DO NOT lost when I simply execute `$ docker start elasticsearch.container` command?

Copied from original issue: ansible/ansible-modules-core#1938

**Comment by ansibot** [comment]
From @ansibot on 2015-08-19T13:38:38Z



@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansibot Greetings! Thanks for taking the time to open this issue. In order for the community to handle your issue effectively, we need a bit more information. 

Here are the items we could not find in your description:
- issue type
- ansible version
- component name

Please set the description of this issue with this template:
https://raw.githubusercontent.com/ansible/ansible/devel/.github/ISSUE_TEMPLATE.md

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (104/100 (104%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext3.iad.github.net ([192.30.252.194]:51369 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.89)
	(envelope-from <noreply@github.com>)
	id 1drnDe-000hrW-Nq
	for greg@biomassiv.es; Tue, 12 Sep 2017 10:35:22 -0500
Date: Tue, 12 Sep 2017 08:27:47 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1505230482;
	bh=sz0VoAzbxhOyHONip20Hkm/PSZO4W9ffKetWKjmZJLQ=;
	h=From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=pqu+ZrPZ38/1qkHW4zCkMVJMOa0oEdOVC1j53XSmY7imWzBYngRbLSpQT3CgRLZuL
	 sOq65ykKisBhvmPZppVMCFZRPpXiAwZnfr3r6QGTWf3cW0yPyzJeF9HiTxo8D4HTSS
	 z3GCOJEI+CGl+oQEo5SV3DzuVmvUjoAG9f7DHuho=

**Comment by ansibot** [comment]
cc @dusdanig @joshuaconner
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by gundalow** [comment]
The `cloud/docker/_docker.py` no longer exists in Ansible 2.5.0.

For information about the newer Docker modules to use, please see http://docs.ansible.com/ansible/devel/scenario_guides/guide_docker.html

If after reviewing the newer modules you believe this issue/PR would still be useful addition to the newer modules, please create a *new* issue/PR against the new module and reference this issue/PR in the comments.

Closing this. Any comments on here will be ignored.
